src/bika/lims/browser/publish/emailview.py

```diff
@@ -300,7 +299,14 @@ def email_body(self):
         body = self.request.get("body", None)
         if body is not None:
             return body
-        return self.context.translate(_(self.email_template(self)))
+        setup = api.get_setup()
+        body = setup.getEmailBodySamplePublication()
+        template_context = {
+            "client_name": self.client_name,
+        }
```

Comment:
Better call `self.render_email_template` without a value set for parameter `template_context` and let that function to set the `client_name` if not provided (that function initializes a `template_context` with the "recipients" already)

Reply:
The `recipients` variable can be replaced after the email was sent (remember the checkboxes to disable a specific recipient).
Calling it without `recipients` would remove the `$recipients` from the template.